llvm/lib/TargetParser/ARMTargetParser.cpp

```diff
@@ -600,11 +601,17 @@ StringRef ARM::getARMCPUForArch(const llvm::Triple &Triple, StringRef MArch) {
   llvm_unreachable("invalid arch name");
 }
 
-void ARM::PrintSupportedExtensions() {
+void ARM::PrintSupportedExtensions(std::map<StringRef, StringRef> llvmDescMap) {
   outs() << "All available -march extensions for ARM\n\n";
   for (const auto &Ext : ARCHExtNames) {
     // Extensions without a feature cannot be used with -march.
-    if (!Ext.Feature.empty())
-      outs() << '\t' << Ext.Name << "\n";
+    if (!Ext.Feature.empty()) {
```

Comment:
Remove a level of indentation by making this:
```
if (Ext.Feature.empty())
  continue;

outs() << '\t' << Ext.Name << "\n";
<...>
```

Reply:
https://llvm.org/docs/CodingStandards.html#use-early-exits-and-continue-to-simplify-code